docs/cloud/dev/spryker-cloud-commerce-os/preparation-for-going-live.md

```diff
@@ -9,6 +10,13 @@ redirect_from:
   - /docs/en/preparation-for-going-live
 ---
 
+{% info_block warningBox "Do not risk your Go-Live!" %}
+
+The preparation steps listed here are mandatory because they are critical to the success of your go-live. We expect every Spryker customer and partner to complete the list as described below. We can't assist you with problems related to your go-live that are caused by these steps not being completed in time. Ensure that your project plan contains tasks related to the go-live checklist and allows enough time for them to be completed. 
```

Comment:
We cannot expect every customer to be on the latest versions of all modules, for example.

Other points in the document are also questionable, and I would suggest to split them into MUST and RECOMMENDED.

And this phrase `We can't assist you with problems related to your go-live` is a bit too hard.

Reply:
@profuel, what point are you mentioning exactly?
@ThomasLehnerSpryker

Reply:
I will talk to Thomas about this piece next year.

Reply:
@profuel any updates on this PR?